lib/gui/lvgl/Kconfig.input

```diff
@@ -24,6 +24,24 @@ config LVGL_POINTER_KSCAN_MSGQ_COUNT
 	help
 	  Maximum number of items in the keyboard scan message queue.
 
+config LVGL_POINTER_KSCAN_SWAP_XY
+	bool "Swap keyboard scan X,Y axes"
+	help
+	  Swap keyboard scan X,Y axes. This option can be used to align keyboard
```

Comment:
Without a background knowledge, "keyboard scan" is confusing, could it be replaced by "touch scan" or "input scan"?

Reply:
I agree "Keyboard scan" is a confusing name, but other Kscan settings use it, e.g. `"Keyboard scan message queue count maximum"`. So I thought it was better to be consistent until one day we have a specific API for touch devices. What is your suggestion?

Reply:
Seen in this light, "Keyboard scan" is less bad.